routes/news.js

```diff
@@ -25,5 +25,22 @@ router.get('/', async (req, res, next) => {
     }
 })
 
+// GET NEW BY ID 
+router.get('/:id', async (req, res)=> {
+    const { id } = req.params;
+    try {
+    const myNew = await Entry.findOne({
```

Comment:
El singular de news, es news, en ingles no existe singular de news, lo que podes poner es "singleNews" "oneNews" o simplemente tratar de evitar el tema totalmente y ponerle "result" a la variable y listo

Reply:
The word “news” in English is considered singular and uncountable. So we use the singular forms of verbs, like is and was: the news is on channel 5, the news was surprising. Don’t use are or were.

So if the word news is singular and uncountable, how can we talk about more than one piece of news? Well, that’s one possibility – use the word “piece” or “bit” – for example, “I have three pieces/bits of bad news for you” if you want to talk about three distinct topics.